examples/workers/vars-workers-instance.auto.tfvars

```diff
@@ -4,8 +4,10 @@
 worker_pools = {
   oke-vm-instance = {
     description = "Self-managed Instances",
-    mode        = "instance", size = 1,
-    node_labels = { "keya" : "valuea", "keyb" : "valueb" },
+    mode        = "instance",
+    size        = 1,
+    node_labels = { "keya" : "valuea",
```

Comment:
Minor: indentation/newlines for consistency in `node_labels` here and in `examples/workers/vars-workers-instancepool.auto.tfvars`.

Reply:
Might also replace the `:` with `=` for consistency - I've noticed `terraform fmt` does a better job with them also.

Reply:
OK, I fixed the formatting and replaced `:` with `=`